assets/css/main.css

```diff
@@ -1758,4 +1758,10 @@ body {
   -webkit-justify-content: space-between;
   -ms-flex-pack: justify;
   justify-content: space-between;
-}
\ No newline at end of file
+}
+
+/*custom phalcon link*/
+
+@media (max-width: 767px) {
+  .phalcon-blog__item-article-block {
+    overflow-x: auto; } }
```

Comment:
```suggestion
    @media (max-width: 767px) {
    .phalcon-blog__item-article-block {
        overflow-x: auto;
    }
}
```

Reply:
Thank you, corrected the style above, I try to adhere to the writing style that is above